postinstall/Makefile

```diff
@@ -0,0 +1,7 @@
+TOP=..
+include $(TOP)/configure/CONFIG
+
+install:
+	ln -s $(shell realpath $(INSTALL_HOST_BIN))/Camera /usr/local/bin/
```

Comment:
Btw can you still run `make clean` with this simple makefile? Do the includes handle it?

Reply:
> Could this use `TOP` instead of `realpath`?

Yep. That seems better indeed.

> Btw can you still run `make clean` with this simple makefile?

Hmm. I haven't analyzed it. I bet we would have broken links around. Probably it will require a `clean` target to `unlink` them.

Reply:
>> Could this use TOP instead of realpath?
>
> Yep. That seems better indeed.

Hmm. Actually, `TOP` and `INSTALL_LOCATION` are relative paths to the current folder when we reach this target (`INSTALL_LOCATION` becomes a copy of `TOP` as it is not defined in the `CONFIG_SITE`). Here are the available variables:

```Makefile
TOP = ..

INSTALL_LOCATION = ..
INSTALL_ABSOLUTE = /opt/ad-aravis-epics-ioc
FINAL_LOCATION = /opt/ad-aravis-epics-ioc
INSTALL_LOCATION_BIN = ../bin
EPICS_HOST_ARCH = linux-x86_64
INSTALL_HOST_BIN = ../bin/linux-x86_64

IOCS_APPL_TOP = /opt/ad-aravis-epics-ioc
```

I thought it would be okay to use `FINAL_LOCATION` (or `INSTALL_ABSOLUTE`), which are absolute and undocumented, along with `INSTALL_HOST_BIN`, but it turns out it does not work well due to `INSTALL_HOST_BIN` being defined based on the relative `TOP`. That is, `$(INSTALL_ABSOLUTE)/$(INSTALL_HOST_BIN)/Camera` does not point to the correct binary path.

For me, either we use my original suggestion or `$(FINAL_LOCATION)/bin/$(EPICS_HOST_ARCH)/Camera`. I think using `FINAL_LOCATION` might be more appropriate here, as this is really the place the build system expects the binary to go (even though [it does not finish the job on its own][1]).

[1]: https://epics.anl.gov/tech-talk/2023/msg01512.php

Reply:
Using `FINAL_LOCATION` seems like a good choice.